manifests/system_probe.pp

```diff
@@ -4,6 +4,7 @@
   Optional[String] $log_file = undef,
   Optional[String] $sysprobe_socket = undef,
   Optional[Boolean] $enable_oom_kill = false,
+  Optional[Boolean] $runtime_security_config = false,
```

Comment:
Can you make this a `Hash` instead of a `Boolean` and set `runtime_security_config => $runtime_security_config`? Saying this because there are other options that can be set in addition to `enabled`.

Reply:
Would you mind making this change? Not completely sure what you mean